src/install/lockfile.zig

```diff
@@ -2899,7 +2900,7 @@ pub const Package = extern struct {
                             }
                         }
                     },
-                    else => {},
```

Comment:
are you sure this is unreachable?

Reply:
We are returning `error.InvalidPackageJSON` from an identical loop above &minus; unless `json` gets modified mid-way it will never get here.